etc/eth-contracts/contracts/AdminControlled.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: CC0 1.0 Universal
```

Comment:
@mfornet @sept-en This is incorrect, BTW. It should be just `SPDX-License-Identifier: CC0-1.0`.

Reply:
Fixed in 3b6bc3753616c65bd5160e4d88bec1c818b5c275.